windows/win_nssm.ps1

```diff
@@ -22,69 +24,23 @@ $ErrorActionPreference = "Stop"
 # POWERSHELL_COMMON
 
 $params = Parse-Args $args;
+
 $result = New-Object PSObject;
 Set-Attr $result "changed" $false;
 
+$name = Get-Attr $params "name" -failifempty $true
+$state = Get-Attr $params "state" "present" -validateSet "present", "absent", "started", "stopped", "restarted" -resultobj $result
```

Comment:
I would change this to:
```
$state = Get-Attr $params "state" -validateSet "present", "absent", "started", "stopped", "restarted" -default "present" -resultobj $stateResult
```
That way you are explicitly stating the result object and the default state.